### DISCUSSION TITLE
Secure URLs

### DISCUSSION THREAD

**Discussion by louietan** [nfriedly/node-unblocker#224] [Ideas]
Just discovered this project recently, and tried it to bypass the Great Firewall in my country, it worked great.

But my main use is at my workplace to bypass URL filtering. node-unblocker doesn't work in this case, because they uses self-signed certificates, which makes HTTPS meaningless. They are still able to decrypt and check the URL because they are in fact the "man-in-the-middle".

Sorry to pop this question again, I've read #184 , but is it possible to add this feature and make it optional? 
Encrypted URLs may have limitations, but limited is still better than blocked.

**Comment by louietan** [comment]
Oops, sorry again, I should've looked through the issue list more thoroughly.
As it's been already discussed in #141 , this discussion becomes unnecessary, but I can't find a way to close or delete this discussion.
Anyway, I appreciate your work on this project. I've spent some time experimenting the implementation and it was tricky.

**Reply by nfriedly** [reply]
No worries; I thought I had replied to your first post, but it doesn't look like it..

Anyways, I can go ahead and delete this discussion if you'd like.
